enterprise/reporting/flusher/db/postgres_test.go

```diff
@@ -0,0 +1,54 @@
+package db
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetStart(t *testing.T) {
+	db, mock, err := sqlmock.New()
```

Comment:
Using an actual Postgres instance for our testing here would be preferable.

Using github.com/rudderlabs/rudder-go-kit/testhelper/docker/resource/postgres

Reply:
@lvrach I have added a [test](https://github.com/rudderlabs/rudder-server/pull/4823/files#diff-fa63fee478295099e0dc2a7ed417aa52f1a50f411d4cdedfa56452b97c14ccf1) which does the test of entire Runner+Flusher flow using postgres docker. I think it should be enough. Postgres coverage is over 80%

Reply:
@lvrach I have removed the db package. Earlier I thought may be can can db agnostic implementation of the flusher. Since, we are making use of postgres queries directly inside aggregator for performance reasons, I think we  can remove the db interface. Our flusher now works only with postgres